### PR TITLE
*: improve a batch of summary in metadata

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-connection-parameters.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Connection Pools and Connection Parameters
+summary: This document explains how to configure connection pools and parameters for TiDB. It covers connection pool size, probe configuration, and formulas for optimal pool sizing. It also discusses JDBC API usage and MySQL Connector/J parameter configurations for performance optimization.
```
Comment (ran-huang) [inline develop/dev-guide-connection-parameters.md:3]:
```suggestion
```

File: develop/dev-guide-playground-gitpod.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Gitpod
+summary: Gitpod provides a cloud-based development environment that is automated and pre-configured. It allows you to develop, run, and test code directly in the browser without any local configurations.
```
Comment (ran-huang) [inline develop/dev-guide-playground-gitpod.md:3]:
```suggestion
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Bookshop Example Application
+summary: Bookshop is an online bookstore app where you can buy books, rate them, and import table structures and data via TiUP or TiDB Cloud. The tables include books, authors, users, ratings, book_authors, and orders. The database initialization script is also provided.
```
Comment (ran-huang) [inline develop/dev-guide-bookshop-schema-design.md:3]:
```suggestion
```

File: dashboard/dashboard-ops-reverse-proxy.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Use TiDB Dashboard behind a Reverse Proxy
 aliases: ['/docs/dev/dashboard/dashboard-ops-reverse-proxy/']
+summary: TiDB Dashboard can be safely exposed using a reverse proxy. To do this, get the actual TiDB Dashboard address and configure the reverse proxy using either HAProxy or NGINX. You can also customize the path prefix for the TiDB Dashboard service. To enhance security, configure a firewall.
```
Comment (ran-huang) [inline dashboard/dashboard-ops-reverse-proxy.md:4]:
```suggestion
```

File: benchmark/v3.0-performance-benchmarking-with-sysbench.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 title: TiDB Sysbench Performance Test Report -- v3.0 vs. v2.1
 aliases: ['/docs/dev/benchmark/v3.0-performance-benchmarking-with-sysbench/','/docs/dev/benchmark/sysbench-v4/']
+summary: TiDB v3.0 and v2.1 were compared in an OLTP scenario test in June 2019 in Beijing. The test ran on AWS EC2 using CentOS-7.6.1810-Nitro image. Sysbench was used to import 16 tables with 10,000,000 rows each. TiDB v3.0 outperformed v2.1 in all tests, with higher QPS and lower latency. Configuration changes in v3.0 contributed to the improved performance.
```
Comment (qiancai) [inline benchmark/v3.0-performance-benchmarking-with-sysbench.md:4]:
```suggestion
summary: TiDB v3.0 outperformed v2.1 in all tests, with higher QPS and lower latency. Configuration changes in v3.0 contributed to the improved performance.
```

File: benchmark/v5.1-performance-benchmarking-with-tpcc.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TiDB TPC-C Performance Test Report -- v5.1.0 vs. v5.0.2
+summary: TiDB v5.1.0 TPC-C performance is 2.8% better than v5.0.2. Test environment AWS EC2. Hardware PD m5.xlarge (3), TiKV i3.4xlarge (3), TiDB c5.4xlarge (3), TPC-C c5.9xlarge (1). Software PD, TiDB, TiKV v5.0.2 and v5.1.0, TiUP 1.5.1. Parameter configuration is the same for both versions. Test plan includes deployment, database creation, data import, stress testing, and result extraction.
```
Comment (qiancai) [inline benchmark/v5.1-performance-benchmarking-with-tpcc.md:3]:
```suggestion
summary: TiDB v5.1.0 TPC-C performance is 2.8% better than v5.0.2. Parameter configuration is the same for both versions. Test plan includes deployment, database creation, data import, stress testing, and result extraction.
```

File: benchmark/v5.4-performance-benchmarking-with-tpch.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TiDB TPC-H Performance Test Report -- v5.4 MPP mode vs. Greenplum 6.15.0 and Apache Spark 3.1.1
+summary: TiDB v5.4 MPP mode outperforms Greenplum 6.15.0 and Apache Spark 3.1.1 in TPC-H 100 GB performance test. TiDB's MPP mode is 2-3 times faster. Test environment includes hardware and software prerequisites, and parameter configurations for each solution. Test results show TiDB v5.4 has significantly lower query execution times compared to Greenplum and Apache Spark.
```
Comment (qiancai) [inline benchmark/v5.4-performance-benchmarking-with-tpch.md:3]:
```suggestion
summary: TiDB v5.4 MPP mode outperforms Greenplum 6.15.0 and Apache Spark 3.1.1 in TPC-H 100 GB performance test. TiDB's MPP mode is 2-3 times faster. Test results show TiDB v5.4 has significantly lower query execution times compared to Greenplum and Apache Spark.
```

File: benchmark/v6.1-performance-benchmarking-with-tpcc.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TiDB TPC-C Performance Test Report -- v6.1.0 vs. v6.0.0
+summary: TiDB v6.1.0 TPC-C performance is 2.85% better than v6.0.0. Test environment AWS EC2, with hardware and software configurations. TiDB and TiKV parameter configurations are the same for both versions. HAProxy is used for load balancing. Test data preparation involves database deployment, data import, and stress testing. Results show performance improvement across different thread counts.
```
Comment (qiancai) [inline benchmark/v6.1-performance-benchmarking-with-tpcc.md:3]:
```suggestion
summary: TiDB v6.1.0 TPC-C performance is 2.85% better than v6.0.0. TiDB and TiKV parameter configurations are the same for both versions. HAProxy is used for load balancing. Results show performance improvement across different thread counts.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: benchmark/benchmark-sysbench-v6.0.0-vs-v5.4.0.md [hunk 9]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TiDB Sysbench Performance Test Report -- v6.0.0 vs. v5.4.0
+summary: TiDB v6.0.0 shows a 16.17% improvement in read-write workload performance compared to v5.4.0. Other workloads show similar performance between the two versions. The test environment includes AWS EC2 instances and the software versions used are PD v5.4.0 and v6.0.0, TiDB v5.4.0 and v6.0.0, TiKV v5.4.0 and v6.0.0, and Sysbench 1.1.0-df89d34. The parameter configurations for TiDB, TiKV, and global variables are also provided. The test plan involves deploying TiDB, importing tables, executing statements, and performing stress tests via HAProxy. Test results show performance comparisons for point select, update non-index, update index, and read-write workloads.
```
Comment (qiancai) [inline benchmark/benchmark-sysbench-v6.0.0-vs-v5.4.0.md:3]:
```suggestion
summary: TiDB v6.0.0 shows a 16.17% improvement in read-write workload performance compared to v5.4.0. Other workloads show similar performance between the two versions. Test results show performance comparisons for point select, update non-index, update index, and read-write workloads.
```

File: benchmark/benchmark-sysbench-v5.4.0-vs-v5.3.0.md [hunk 10]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TiDB Sysbench Performance Test Report -- v5.4.0 vs. v5.3.0
+summary: TiDB v5.4.0 shows improved performance of 2.59% to 4.85% in write-heavy workloads compared to v5.3.0. The test environment includes AWS EC2 with specific hardware and software configurations. The test plan involves deploying TiDB, using Sysbench to import tables, and performing stress tests. Results show performance improvements in point select, update non-index, update index, and read write scenarios.
```
Comment (qiancai) [inline benchmark/benchmark-sysbench-v5.4.0-vs-v5.3.0.md:3]:
```suggestion
summary: TiDB v5.4.0 shows improved performance of 2.59% to 4.85% in write-heavy workloads compared to v5.3.0. Results show performance improvements in point select, update non-index, update index, and read write scenarios.
```

File: benchmark/benchmark-sysbench-v5-vs-v4.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TiDB Sysbench Performance Test Report -- v5.0 vs. v4.0
+summary: TiDB v5.0 outperforms v4.0 in Sysbench performance tests. Point Select performance improved by 2.7%, Update Non-index by 81%, Update Index by 28%, and Read Write by 9%. The test aimed to compare performance in the OLTP scenario using AWS EC2. Hardware and software configurations were specified for both versions. Test plan included data preparation and execution. Test results were presented in tables and graphs.
```
Comment (qiancai) [inline benchmark/benchmark-sysbench-v5-vs-v4.md:3]:
```suggestion
summary: TiDB v5.0 outperforms v4.0 in Sysbench performance tests. Point Select performance improved by 2.7%, Update Non-index by 81%, Update Index by 28%, and Read Write by 9%. The test aimed to compare performance in the OLTP scenario using AWS EC2. Test results were presented in tables and graphs.
```

**Pull request merged**